src/client/app/components/BarControlsComponent.tsx

```diff
@@ -89,6 +89,7 @@ export default function BarControlsComponent() {
 			dispatch(graphSlice.actions.updateBarDuration(moment.duration(value, 'days')));
 		}
 	};
+	const translate = useTranslate();
```

Comment:
While this comment is here it applies to many files changed. The location of this line varies by file and is not, from my perspective, in the ideal location in the code. It is similar to ``const dispatch = useAppDispatch();`` that is at/near the top of the function. I think that is a good location for this type of code. I think all of these lines should be there (or in the block defining functions as in some files) or near the top of the function if dispatch is not present. If you feel otherwise then please let me know.

Reply:
@huss , I too see the same type of failure.  After debugging, I see that placing `const translate = useTranslate();` at the top of the function was crashing in some components. However placing it somewhere below was rectifying the error. I was unable to technically decode why this was happening.
On the other hand, the translate changes in some components was resulting in error and that is why some admin pages were crashing. However in this case, altering the location of `const translate = useTranslate();` did not work out.